### PR TITLE
Issue #SB-29145 feat: Added Question/QuestionSet Copy API Documentation

### PR DESCRIPTION
**Pull request by joffinjoy**

### CODE REVIEW

File: apis/questionapi/question.yaml [hunk 1]
```diff
@@ -3209,6 +3209,140 @@ paths:
...
       security:
         - bearer: []
+  /questionset/v1/copy/{QuestionSet_Id}:
+    post:
+      description: |-
+        This API is associated with copying questionset on the Sunbird Platform.
+          - **_/Copy/_** endpoint executes the "Copy QuestionSet" request based on the parameters provided as metadata in the request body.
+          - The endpoint for **Copy QuestionSet** is `/questionset/v1/copy`
+          - It points to assessment-service - `/questionset/v4/copy`
+          - {...}refers to a section of copy endpoint URL path that needs to be replaced with the appropriate ID.
```
Comment (anisha147) [inline apis/questionapi/question.yaml]: Add a space after {...}

File: apis/questionapi/question.yaml [hunk 2]
```diff
@@ -3209,6 +3209,140 @@ paths:
...
+          - Mandatory fields cannot be null or empty.
+      summary: Copy QuestionSet
+      tags:
+        - QuestionSet APIs
+      operationId: Copy QuestionSet
+      parameters:
+        - in: path
+          name: QuestionSet_Id
+          required: true
+          description: Append a valid QuestionSet Id To the Request URL
```
Comment (anisha147) [inline apis/questionapi/question.yaml]: Please lowercase "To"

File: apis/questionapi/question.yaml [hunk 3]
```diff
@@ -3209,6 +3209,140 @@ paths:
...
+                  msgid: null
+                  err: ERR_INVALID_REQUEST
+                  status: failed
+                  errmsg: Please provide valid value for List(createdFor)
+                responseCode: CLIENT_ERROR
+                result:
+                  messages: null
+        '404':
+          description: >-
+            RESOURCE NOT FOUND,The **Copy Question Set** operation failed !The possible
```
Comment (anisha147) [inline apis/questionapi/question.yaml]: Please rectify the formatting in the description.

File: apis/questionapi/question.yaml [hunk 4]
```diff
@@ -3446,6 +3580,155 @@ paths:
       deprecated: false
       security:
         - bearer: []
+  /question/v1/copy/{Question_Id}:
+    post:
+      description: |-
+        This API is associated with copying Questions on the Sunbird Platform .
```
Comment (anisha147) [inline apis/questionapi/question.yaml]: Please remove space before the end punctuation

File: apis/questionapi/question.yaml [hunk 5]
```diff
@@ -3446,6 +3580,155 @@ paths:
...
+          - **_/Copy/_** endpoint executes the "Copy Question" request based on parameters provided as metadata in the request body.
+          - The endpoint for **Copy Question** is `/question/v1/copy`
+          - It points to assessment-service - `/question/v4/copy`
+          - {...}refers to a section of review endpoint URL path that needs to be replaced with the appropriate ID.
```
Comment (anisha147) [inline apis/questionapi/question.yaml]: Please add space after {...}

File: apis/questionapi/question.yaml [hunk 6]
```diff
@@ -3446,6 +3580,155 @@ paths:
...
+          - Mandatory fields cannot be null or empty.
+      summary: Copy Question
+      tags:
+        - Question APIs
+      operationId: Copy Question
+      parameters:
+        - in: path
+          name: Question_Id
+          required: true
+          description: Append a valid Question ID To the Request URL
```
Comment (anisha147) [inline apis/questionapi/question.yaml]: Please lowercase "To"

File: apis/questionapi/question.yaml [hunk 7]
```diff
@@ -3446,6 +3580,155 @@ paths:
...
+                      msgid: null
+                      err: ERR_INVALID_REQUEST
+                      status: failed
+                      errmsg: Question With Visibility Parent Cannot Be Copied Individually!
+                    responseCode: CLIENT_ERROR
+                    result:
+                      messages: null
+        '404':
+          description: >-
+            RESOURCE NOT FOUND,The **CopyQuestion** operation failed!The possible
```
Comment (anisha147) [inline apis/questionapi/question.yaml]: Please rectify the formatting in the description

File: apis/questionapi/question.yaml (hunk 3 above)
Comment (joffinjoy) [reply apis/questionapi/question.yaml]: Did you mean the lack of space after FOUND & space between failed and ! ?

Comment (anisha147) [reply apis/questionapi/question.yaml]: Yes, exactly.

**Pull request closed without merging**